feat(k8s collection): document how to use only docker hub registry (#4003)
SumoLogic/sumologic-documentation: docs/send-data/kubernetes/best-practices.md

    ## See https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/fips.md for more information.
    addFipsSuffix: false
```

> ## Use only Docker Hub registry

Author: ```suggestion
## Use containers only from Docker Hub registry
```

    ## See https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/fips.md for more information.
    addFipsSuffix: false
```

## Use containers only from Docker Hub registry

> If for some reason you can use only DockerHub as docker registry, we are suggesting to use the following configuration:

Reviewer: ```suggestion
If DockerHub is the only available Docker registry option, we recommend the following configuration:
```

  kubeRBACProxy:
    image:
      repository: docker.io/bitnami/kube-rbac-proxy
      # Ensure the tag is the same like in Sumo Logic Kubernetes Collection Chart, but without `v` prefix
      tag: 0.15.0
tailing-sidecar-operator:
  kubeRbacProxy:
    image:
      repository: docker.io/bitnami/kube-rbac-proxy
      # Ensure the tag is the same like in Sumo Logic Kubernetes Collection Chart, but without `v` prefix
      tag: 0.11.0
  operator:
    image:
      repository: docker.io/sumologic/tailing-sidecar-operator
  sidecar:
    image:
      repository: docker.io/sumologic/tailing-sidecar
telegraf-operator:
  image:
    # Ensure the image tag is the same like in Sumo Logic Kubernetes Collection Chart
    sidecarImage: docker.io/library/telegraf:1.21.2
    # There is no recommended Docker Hub image for Telegraf Operator
    # repository: 
kube-prometheus-stack:
  prometheus-node-exporter:
    image:
      repository: docker.io/prom/node-exporter
  kube-state-metrics:
    image:
      repository: docker.io/bitnami/kube-state-metrics
      # Ensure the tag is the same like in Sumo Logic Kubernetes Collection Chart, but without `v` prefix
      tag: 2.7.0
```

> :::note
> 
> Not all of the images are maintained by Sumo Logic and not all the vendors provides their images on Docker Hub.
> We suggest the most similar images but we do not guarantee that they will work exactly the same, but they should.
> 
> :::

Reviewer: ```suggestion
:::note
While we recommend the most similar images, it's important to note that not all images are maintained by Sumo Logic, and some vendors may not provide their images on Docker Hub. As a result, while the suggested images should work similarly, we cannot guarantee identical functionality.
:::
```

Author: I removed this note as we clone all of the images right now

Reviewer: thank you!
